### PR TITLE
Created the home screen

### PR DESCRIPTION
**Pull request by logickoder**
image

### CODE REVIEW

File: app/src/main/java/dev/logickoder/jobfinder/home/JobSearch.kt [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                content = {
+                    Row(
+                        content = {
+                            Icon(
+                                modifier = Modifier.padding(end = paddingSmall() / 3),
+                                imageVector = Icons.Filled.Search,
+                                contentDescription = null,
+                            )
+                            Text(
+                                text = "Search any job...",
```
Comment (LuuNgocLan) [inline app/src/main/java/dev/logickoder/jobfinder/home/JobSearch.kt]: Pls put hardcode to the resource file

File: app/src/main/java/dev/logickoder/jobfinder/app/widget/Rating.kt [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    textColor: Color = MaterialTheme.colorScheme.onSurface,
+) {
+    Row(
+        modifier = modifier,
+        verticalAlignment = Alignment.CenterVertically,
+        content = {
+            Icon(
+                imageVector = Icons.Filled.Star,
+                contentDescription = null,
+                tint = Color(0xFFF8D048),
```
Comment (LuuNgocLan) [inline app/src/main/java/dev/logickoder/jobfinder/app/widget/Rating.kt]: pls define color

File: app/src/main/java/dev/logickoder/jobfinder/home/HomeScreen.kt [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+                        ),
+                    )
+                    JobSearch(
+                        modifier = Modifier.padding(
+                            horizontal = padding(),
+                        )
+                    )
+                    Row(
+                        modifier = Modifier
+                            .padding(horizontal = padding())
```
Comment (LuuNgocLan) [inline app/src/main/java/dev/logickoder/jobfinder/home/HomeScreen.kt:65]: pls check the padding right

File: app/src/main/java/dev/logickoder/jobfinder/home/HomeTopBar.kt [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+import dev.logickoder.jobfinder.app.theme.JobFinderTheme
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun HomeTopBar(modifier: Modifier = Modifier) {
+    CenterAlignedTopAppBar(
+        modifier = modifier,
```
Comment (LuuNgocLan) [inline app/src/main/java/dev/logickoder/jobfinder/home/HomeTopBar.kt:27]: you missed padding the horizontal of the tab bar.

**Review by LuuNgocLan — COMMENTED** [review]
Pls check comments

File: app/src/main/java/dev/logickoder/jobfinder/home/HomeTopBar.kt (hunk 4 above)
Comment (logickoder) [reply app/src/main/java/dev/logickoder/jobfinder/home/HomeTopBar.kt:27]: Yeah, it didn't look so visually pleasing to me, so I removed it in the first place. But I've added it back though

**Comment by logickoder** [comment]
image

**Comment by LuuNgocLan** [comment]
LGTM

**Pull request merged**